wiki/markdown/Running-rules-on-objects.md

```diff
@@ -4,17 +4,26 @@
 |------|---------------------|
 |POST |/rules/rule-check|
 |POST | /rules/rule-check/aggregation|
+|GET | /rules/rule-check/testRulePageEnabled |
 
-Before deploying a new instance of Eiffel Intelligence it is good to test the 
-desired rules that they give the desired outcome. Therefore it is possible to 
-test the rules on a test instance of Eiffel Intelligence. This means that this 
-instance needs to be dedicated to only testing rules and it can be one per 
-company, organization or department. The property controlling this feature 
+
+Before deploying a new instance of Eiffel Intelligence it is good to test the
+desired rules that they give the desired outcome. Therefore it is possible to
+test the rules on a test instance of Eiffel Intelligence. This means that this
+instance needs to be dedicated to only testing rules and it can be one per
+company, organization or department. The property controlling this feature
 should be set to true:
 
     testaggregated.enabled: true
 
-In this way every user testing will get its own test space that will be removed 
+## Check if testRules are enabled on the selected instance
```

Comment:
This subsection maybe should be placed after this sentence: "A GUI is also implemented in [Eiffel Intelligence Frontend] ..."

Reply:
Done. Moved the paragraph about the GUI for a better flow in the text.